frog/imports/api/uploads.js

```diff
@@ -1,9 +1 @@
-// @flow
-
-import { FS } from 'meteor/cfs:base-package';
-
-export const Uploads = new FS.Collection('uploads', {
-  stores: [new FS.Store.FileSystem('uploads')]
-});
-
-Uploads.allow({ insert: () => true });
+export const Uploads = { insert: () => {} };
```

Comment:
Putting this here for now to make Flow happy. Will rethink a bit how to do this to make sure it works both in development and production.

Reply:
ok